### DISCUSSION TITLE
Can't access web interface

### DISCUSSION THREAD

**Discussion by johnwillyn** [aqualinkd/AqualinkD#211] [General]
I just discovered this project and it is exactly what I have been searching for to help me diagnose issues with my Jandy RS panel.  Awesome piece of work.  Thanks!  I hope I can contribute to the community.

I have everything installed on a Raspberry Pi 4 and running but have one minor issue with accessing the web interface.

I can access the web interface in the browser running over a VPN connection with either localhost or the ip address, but when I try to access it from my laptop using the ip address, it does not respond and the browser times out.

The Pi and my laptop are on the same WiFi, and I have a number of other apps on other Pi SBC where I access node servers, but I can't seem the web server on the aqualinkd Pi.

I must be missing something simple.  Any suggestions welcome!

JohnL

**Comment by sfeakes** [comment]
I don't think you are missing anything.  AqualinkD doesn't use any form of security or limit what IP's have access.
Since VPN works but Local doesn't, I would say your laptop and Pi are on a different subnet and you have not bridges those networks on your router, or you have something in the raspberry blocking external access (but allowing internal access) to the port AqualinkD is using.  In short it sounds like a network problem.

Easy way to tell on the subnet issue would be to look at the IP address on your laptop and raspberry, if the first 2 groups of numbers are the same, you are on the same subnet and that's not the problem.

Running something like the below on the Pi would tell you what's listening on that port.
`netstat -tnlp | grep -w 80`  <- change 80 to what ever port you are using.
If you see anything other that aqualinkd, or any ip other than 0.0.0.0 then something might be blocking in on the Pi.

I can't think of anything else at the moment.

**Comment by johnwillyn** [comment]
Thanks for taking the time to respond sfeakes.

I believe my issue is related to lack of NAT "loopback" or "hairpin" support on the Google Mesh network, which I am now using as my primary router.  I have run into this before when I used a Comcast modem/router in front of my mesh network, but I got rid of that when the disabled access to port forwarding.

I can work around this with the VPN access, so no big deal.

Here is an extended discussion on the Google Community site (in case it helps someone else).

https://www.googlenestcommunity.com/t5/Nest-Wifi/Google-Wifi-Pro-Nat-Loopback-issue/m-p/307040#M34349

As I mentioned, I am very happy to discover this project, and I believe it will help resolve several issues I have had with the Jandy panel (which I will go into in a separate thread).
